handel/src/services/dynamodb/params-schema.json

```diff
@@ -25,7 +25,7 @@
                 "type": {
                     "type": "string",
                     "description": "The 'type' parameter may only be 'String' or 'Number'",
-                    "pattern": "^String|Number$",
+                    "pattern": "^[sS][tT][rR][iI][nN][gG]|[nN][uU][mM][bB][eE][rR]$",
```

Comment:
I changed my mind, I think we only want to allow lowercase (i.e. 'string') or capital-case (i.e. 'String'). It's really supposed to be 'String' according to DynamoDB but the lowercase variant is just a helper to people who forget to put the capital letter in first.

We may also need to look at modifying some code to make sure this will actually work. Dynamo wants 'String' and 'Number', so we'll have to transform 'string' and 'number' prior to passing them into the CloudFormation template.